helpers.json

```diff
@@ -63,18 +113,59 @@
     "name": "canOpenEyeDoors",
     "requires": [
       {"or": [
-        "Missile",
-        "Super"
+        {"and": [
+          "Missile",
+          {"cost":[
+            {"ammo": [
+              {
+                "type": "Missile",
+                "count": 3
+              }
+            ]}
+          ]}        
+        ]},
+        {"and": [
+          "Super",
+          {"cost":[
+            {"ammo": [
+              {
+                "type": "Super",
+                "count": 1
+              }
+            ]}
+          ]}
+        ]}
       ]}
     ]
   },
   {
     "name": "canOpenZebetites",
     "requires": [
-      "Missile",
-      "Super"
-    ],
-    "note": "14 Missiles, 4 Supers"
+      {"or": [
+        {"and": [
+          "Missile",
+          {"cost":[
+            {"ammo": [
+              {
+                "type": "Missile",
+                "count": 14
```

Comment:
It takes 12 missiles. They can regen if you take too long.

Reply:
Went with the info in the notes. Changing.